modules/asg_node_group/main.tf

```diff
@@ -177,12 +177,6 @@ resource "aws_autoscaling_group" "nodes" {
     propagate_at_launch = false
   }
 
-  tag {
```

Comment:
:question: Shouldn't we replace this with the new label, instead of just removing it?

Reply:
Already in place: https://github.com/cookpad/terraform-aws-eks/blob/remove-deprecated-zone-label/modules/asg_node_group/main.tf#L175